mmdet3d/ops/pointnet_modules/point_sa_module.py

```diff
@@ -68,6 +68,8 @@ def __init__(self,
             self.num_point = [num_point]
         elif isinstance(num_point, list) or isinstance(num_point, tuple):
             self.num_point = num_point
+        elif num_point is None:
+            self.num_point = None
```

Comment:
Maybe we need an additional unittest for the condition when num_point == None.

Reply:
Yes, I will add it later